library/std/src/fs.rs

```diff
@@ -2511,9 +2511,10 @@ impl AsInnerMut<fs_imp::DirBuilder> for DirBuilder {
 /// This function will traverse symbolic links to query information about the
 /// destination file. In case of broken symbolic links this will return `Ok(false)`.
 ///
-/// As opposed to the [`Path::exists`] method, this one doesn't silently ignore errors
-/// unrelated to the path not existing. (E.g. it will return `Err(_)` in case of permission
-/// denied on some of the parent directories.)
+/// As opposed to the [`Path::exists`] method, this will only return `Ok(true)` or `Ok(false)`
+/// if the path was _verified_ to exist or not exist. If its existance can neither be confirmed
+/// nor denied, an `Err(_)` will be propegated instead. This can be the case if e.g. listing
```

Comment:
*existence
*propagated

Reply:
Thanks for the catches, should be all set